docs/pages/versions/unversioned/sdk/blur-view.mdx

```diff
@@ -15,6 +15,9 @@ A React component that blurs everything underneath the view. On iOS, it renders
 
 <PlatformsSection emulator android ios simulator web />
 
+> `BlurView` on Android is computationally expensive, so it is recommended not to render more than a few `BlurView` components at once.
+> On Android `BlurView` may render incorrectly during screen transitions made by `react-native-screens`.
```

Comment:
```suggestion
> On Android `BlurView` may render incorrectly during screen transitions made by `react-native-screens`.
```

Let's move this one to the separate "Known Issues" section like for example we done on FaceDetector or ImagePicker pages:
* https://docs.expo.dev/versions/latest/sdk/facedetector/#known-issues
* https://docs.expo.dev/versions/latest/sdk/imagepicker/#known-issues